src/SixLabors.Fonts/StreamFontMetrics.Cff.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using SixLabors.Fonts.Tables.AdvancedTypographic;
+using SixLabors.Fonts.Tables.Cff;
+using SixLabors.Fonts.Tables.General;
+using SixLabors.Fonts.Tables.General.Colr;
+using SixLabors.Fonts.Tables.General.Kern;
+using SixLabors.Fonts.Tables.General.Name;
+using SixLabors.Fonts.Tables.General.Post;
+using SixLabors.Fonts.Unicode;
+
+namespace SixLabors.Fonts
+{
+    /// <content>
+    /// Contains CFF specific methods.
+    /// </content>
+    internal partial class StreamFontMetrics
+    {
+        private static StreamFontMetrics LoadCompactFont(FontReader reader)
+        {
+            // Load using recommended order for best performance.
+            // https://www.microsoft.com/typography/otspec/recom.htm#TableOrdering
+            // 'head', 'hhea', 'maxp', OS/2, 'name', 'cmap', 'post', 'CFF '
+            HeadTable head = reader.GetTable<HeadTable>();
+            HorizontalHeadTable hhea = reader.GetTable<HorizontalHeadTable>();
+            MaximumProfileTable maxp = reader.GetTable<MaximumProfileTable>();
+            OS2Table os2 = reader.GetTable<OS2Table>();
+            NameTable name = reader.GetTable<NameTable>();
+            CMapTable cmap = reader.GetTable<CMapTable>();
+            PostTable post = reader.GetTable<PostTable>();
+            CffTable cff = reader.GetTable<CffTable>(); // TODO: CFF2, VORG
```

Comment:
Should throw a proper `NotSupportedException`, if its CFF2?

Reply:
Throws on attempted CFF2 table load now.